tests/basic/kubevirt_test.go

```diff
@@ -45,3 +47,58 @@ var _ = Describe("{AddNewDiskToKubevirtVM}", func() {
 		AfterEachTest(contexts)
 	})
 })
+
+var _ = Describe("{KubeVirtLiveMigration}", func() {
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("KubeVirtLiveMigration", "Live migrate a kubevirtVM", nil, 0)
+	})
+
+	var contexts []*scheduler.Context
+
+	itLog := "Live migrate a kubevirtVM"
+	It(itLog, func() {
+		log.InfoD(stepLog)
+		appList := Inst().AppList
+		defer func() {
+			Inst().AppList = appList
+		}()
+		Inst().AppList = []string{"kubevirt-multi-pvc"}
+
+		stepLog := "schedule a kubevirtVM"
+		Step(stepLog, func() {
+			for i := 0; i < Inst().GlobalScaleFactor; i++ {
+				taskName := fmt.Sprintf("test-%v", i)
+				contexts = append(contexts, ScheduleApplications(taskName)...)
+			}
+		})
+
+		ValidateApplications(contexts)
+
+		for _, ctx := range contexts {
+			bindMount, err := kubevirt.VerifyBindMount(ctx, Inst().S, Inst().V)
+			log.FailOnError(err, "Failed to verify bind mount")
+			dash.VerifyFatal(bindMount, true, "Failed to verify bind mount")
+		}
+
+		stepLog = "Live migrate the kubevirt VM"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			for _, ctx := range contexts {
+				success, err := kubevirt.StartAndWaitForVMIMigration(ctx, context1.TODO(), Inst().S)
+				log.FailOnError(err, "Failed to live migrate kubevirt VM")
+				dash.VerifyFatal(success, true, "Failed to live migrate kubevirt VM?")
+			}
+		})
+
+		for _, ctx := range contexts {
+			bindMount, err := kubevirt.VerifyBindMount(ctx, Inst().S, Inst().V)
```

Comment:
this is a wrong expectation for the test. Live migration does not move the attachment. So, the migrated VM will be nfs-mounted. If this test is passing something is wrong with VerifyBindMount function.

Reply:
I was just just testing the function and excepted behaviour is for the function to fail so will remove it once done with testing.